addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -133,16 +133,43 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 .o_wprofile_all_users_nav {
     border-width: 1px 0;
 
-    &, .o_wprofile_course_nav_search {
+    &, .o_wprofile_course_nav_search, .o_wprofile_all_users_nav_btn {
         background-color: rgba(white, 0.05);
         border-color: rgba(white, 0.1);
         border-style: solid;
     }
 
-    .o_wprofile_course_nav_search {
+    .o_wprofile_course_nav_search, .o_wprofile_all_users_nav_btn {
         border-width: 0 1px;
     }
 
+    .o_wprofile_all_users_nav_btn {
+        @include media-breakpoint-up(md) {
+            @include o-hover-text-color(white, $gray-800);
+            margin-top: -1px;
+            border-radius: 0;
+            min-height: 35px;
+
+            &:hover {
+                background-color: white;
+            }
+        }
+    }
+
+    .o_wprofile_all_users_nav_btn_container {
+        @include media-breakpoint-down(sm) {
+            ~ .d-md-none {
+                padding-left: 0;
```

Comment:
`o_wprofile_all_users_nav_btn_container` may exist or not,  depending  if you come from forum

Reply:
Why the `media-breakpoint`? ;)

Reply:
We need the `media-breakpoint` anyway because of the next selector, `o_wprofile_all_users_nav_btn`.
```CSS
.o_wprofile_all_users_nav_btn_container {
    @include media-breakpoint-down(sm) {
        ~ .d-md-none {
            padding-left: 0;
        }

        .o_wprofile_all_users_nav_btn {
            @include o-hover-text-color(white, white);
            border-radius: $btn-border-radius;
            background-color: rgba(black, 0.25);
        }
    }
}
```
We may move ` ~ .d-md-none {}` outside, but I'm wondering what's the point of keeping active a rule that affect a not visible element.
Moreover, even if in the future we will decide to change it's visibility (eg. using `.d-lg-none`), `media-breakpoint-down(sm)` will still remain the right scope, because it's the only resolution in which this rule is needed.

Reply:
Ok, what annoys me is targeting `d-xx-none` classes in css... reminds me of the themes styling col-lg-9 classes while they can be resized by the user... :'( Could you not add a different class ?

Reply:
Sure. 
Just let me just know when you have other remarks so I'll jump back on this project ;)

Reply:
@stefanorigano

Reply:
Handled

Reply:
don't understand why this is in website_profile if only used by forum though :'( but no time to change anymore... :/